### ISSUE TITLE
ClassCastException: java.nio.DirectByteBuffer cannot be cast to java.nio.IntBuffer

### ISSUE THREAD

**Issue by nineclue** [bytedeco/javacv#140]
I'm trying to record javafx scene using OpenCVFrameRecorder and got following error message.
```
java.lang.ClassCastException: java.nio.DirectByteBuffer cannot be cast to java.nio.IntBuffer
	at org.bytedeco.javacv.Java2DFrameConverter.copy(Java2DFrameConverter.java:518)
	at org.bytedeco.javacv.Java2DFrameConverter.getFrame(Java2DFrameConverter.java:701)
	at org.bytedeco.javacv.Java2DFrameConverter.getFrame(Java2DFrameConverter.java:666)
	at org.bytedeco.javacv.Java2DFrameConverter.getFrame(Java2DFrameConverter.java:660)
	at org.bytedeco.javacv.Java2DFrameConverter.convert(Java2DFrameConverter.java:59)
	at Test2.recordFrame(Test2.scala:75)
```
Test2.scala :
```scala  
val bImgConverter = new Java2DFrameConverter()
  def recordFrame[A <: Image](i:A) = {
    val bim = SwingFXUtils.fromFXImage(i, null)
    val f = bImgConverter.convert(bim)  // line 75
    recorder.record(f)
  }
```

JavaCv version : 0.11
Scala : 2.10

Any help would be appreciated.

**Comment by saudet** [comment]
That's a known bug that was fixed here: https://github.com/bytedeco/javacv/commit/918c7540482a87be6b6c83b10abad660b551c140 Thanks for reporting!

**Comment by saudet** [comment]
Fix included in version 1.0. Thanks for reporting!
